advanced/Scripts/piholeCheckout.sh

```diff
@@ -111,59 +193,59 @@ checkout() {
         echo "master" > /etc/pihole/ftlbranch
         chmod 644 /etc/pihole/ftlbranch
     elif [[ "${1}" == "core" ]] ; then
-        str="Fetching branches from ${piholeGitUrl}"
+        str="Fetching branches/tags from ${piholeGitUrl}"
         echo -ne "  ${INFO} $str"
         if ! fully_fetch_repo "${PI_HOLE_FILES_DIR}" ; then
             echo -e "${OVER}  ${CROSS} $str"
             exit 1
         fi
-        corebranches=($(get_available_branches "${PI_HOLE_FILES_DIR}"))
+        corerefs=($(get_available_refs "${PI_HOLE_FILES_DIR}"))
```

Comment:
https://www.shellcheck.net/wiki/SC2207

```suggestion
        corerefs=( "$(get_available_refs "${PI_HOLE_FILES_DIR}")" )
```